opencontrail/files/3.0/contrail-analytics-api.conf

```diff
@@ -11,6 +11,8 @@ log_level = SYS_NOTICE
 log_category = 
 log_file = /var/log/contrail/contrail-analytics-api.log
 
+aaa_mode = {{ collector.api.get('aaa_mode', 'no-auth') }}
```

Comment:
This is for @pupapaik to decide, he has access to contrail 3.0.x to test it. This renders only for version 3, so it should not break version 2.